sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -111,9 +116,25 @@ export default function VisualSearchBar({
       return null;
     }
     const options = autocompleteResults.map(result => {
+      const verticalKeys = hideVerticalLinks ? undefined : ['people', 'events'];//result.verticalKeys;
+      const verticalLinks = verticalKeyToNameMapping && verticalKeys?.map(verticalKey => { 
+        return {
+          label: verticalKeyToNameMapping[verticalKey],
+          verticalKey
+        }
+      });
       return {
         value: result.value,
-        display: renderAutocompleteResult(result, cssClasses.resultIconContainer)
+        verticalLinks,
+        render: (onClick: () => void, isOptionFocus: boolean, focusLinkIndex: number) => 
+          renderAutocompleteResult(
+            result,
+            onClick,
+            cssClasses as AutocompleteResultCssClasses,
```

Comment:
do we need this assertion?

Reply:
nope, forgot to remove when refactoring the styling.